src/args/manage-ports/template.sh

```diff
@@ -50,11 +52,12 @@ function wait_for_tcp {
       error "Timeout while waiting for ${host}:${port} to open" \
         && return 1
     else
-      info "Waiting 1 second for ${host}:${port} to open, ${elapsed} seconds in total" \
+      info "Waiting 1 second for ${host}:${port} to open: ${elapsed} seconds in total" \
         && sleep 1 \
         && elapsed="$(("${elapsed}" + 1))" \
         && continue
     fi
+    break
```

Comment:
I think this line breaks the cycle in the first iteration?

Reply:
That's right, my mistake. I added that line for debugging. Will remove it now.